packages/map/src/components/Legend/components/index.scss

```diff
@@ -70,33 +67,44 @@
     }
 
     .legend-container {
-      --space-between-legend-items: 0.6em;
       position: relative;
-      &.legend-padding {
```

Comment:
Similar to the comment made in charts, you could move these sections into core so that its hopefully easier to see the containers your targeting. I realize there are differences on the divs, but it might help normalize them going forward.